check.py

```diff
@@ -353,7 +353,7 @@ def minify_check(wast, verify_final_result=True):
 print '\n[ checking binaryen-shell spec testcases... ]\n'
 
 if len(requested) == 0:
-  BLACKLIST = []
+  BLACKLIST = ['address.wast', 'traps.wast']
```

Comment:
That was needed because before https://github.com/WebAssembly/spec/pull/209 those tests had memory in units of bytes. But that landed, so I'll update this patch and this can hopefully go away.